instant.sh

```diff
@@ -45,8 +45,19 @@ if [ "$TARGET" == "kubernetes" ] || [ "$TARGET" == "k8s" ]; then
         ./healthworkforce/kubernetes/main/k8s.sh destroy
         ./healthworkforce/kubernetes/importer/k8s.sh clean
     elif [ "$COMMAND" == "test" ]; then
-        ./core/test.sh $minikubeIP:$openhimCoreTransactionSSLPort
-        ./healthworkforce/test.sh $minikubeIP:$openhimCoreTransactionSSLPort
+        openhimCoreHostname=$(kubectl get service openhim-core-service -o=jsonpath="{.status.loadBalancer.ingress[*]['hostname', 'ip']}")
+        openhimCoreTransactionSSLPort=$(kubectl get service openhim-core-service -o=jsonpath={.spec.ports[1].port})
+        hostnameLength=$(expr length "$openhimCoreHostname")
+        echo "Host: $openhimCoreHostname"
+        echo "Length: $hostnameLength"
```

Comment:
Is this maybe testing output or do you want this here?

Reply:
Whoops. Testing :(